tools/sggcov2lcov/tools.go

```diff
@@ -0,0 +1,70 @@
+package sggcov2lcov
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	name    = "gcov2lcov"
+	version = "1.0.6"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	coverFile := sg.FromBuildDir("go", "coverage", "go-test.txt")
+	lcovFile := sg.FromBuildDir("go", "coverage", "go-test.lcov")
+	cmd := sg.Command(
+		ctx,
+		sg.FromBinDir(name),
+		append(
+			[]string{
+				"-infile=" + coverFile,
+				"-outfile=" + lcovFile,
+			},
+			args...,
+		)...,
+	)
+	goroot := sg.Output(sg.Command(ctx, "go", "env", "GOROOT"))
+	cmd.Env = append(cmd.Env, "GOROOT="+goroot)
```

Comment:
Why is this needed? Setting the GOROOT is something one should generally not need

Reply:
It was needed when I ran the command. Not sure about the details, but it was mentioned in the tool docks so there is probably a valid reason: https://github.com/jandelgado/gcov2lcov#goroot

Reply:
Did we try without it? The docs mention it might be needed and it would be great to skip it as it could lead to issues in some setups.

Reply:
Yes, otherwise I would not have added it.

Reply:
Do you have an idea how we can make it optional instead if you prefer?

Reply:
You can let the user simply set the environment variable on the local project. Will the tool even work without it though? I am mostly trying to avoid future issues. Having to set GOROOT feels a little like working against the Go ecosystem but if the tool requires it to work I don't see any harm in keeping it for now. Maybe add a comment on it?